cloudformation/ci.template.js

```diff
@@ -0,0 +1,54 @@
+var cf = require('@mapbox/cloudfriend');
+var package_json = require('../package.json')
+
+module.exports = {
+  AWSTemplateFormatVersion: '2010-09-09',
+  Description: 'user for publishing to s3://mapbox-node-binary/' + package_json.name,
+  Resources: {
+    User: {
+      Type: 'AWS::IAM::User',
+      Properties: {
+        Policies: [
+          {
+            PolicyName: 'list',
+            PolicyDocument: {
+              Statement: [
+                {
+                  Action: ['s3:ListBucket'],
+                  Effect: 'Allow',
+                  Resource: 'arn:aws:s3:::mapbox-node-binary/' + package_json.name
```

Comment:
This policy is wrong and will not work because the resource name is wrong. Should be:

```
          {
            PolicyName: 'list',
            PolicyDocument: {
              Statement: [
                {
                  Action: ['s3:ListBucket'],
                  Effect: 'Allow',
                  Resource: 'arn:aws:s3:::mapbox-node-binary',
                  Condition : {
                    StringLike : {
                      "s3:prefix": [ package_json.name + "/*"]
                    }
                  }
                }
              ]
            }
          },

```

Without this publishing will fail with a weird `Forbidden` error, because `node-pre-gyp` calls `HeadObject` on a (potentially) non-existent file. Only if you have `ListBucket` rights this will return `NotFound` instead of `Forbidden`.